sw/airborne/firmwares/rotorcraft/autopilot_firmware.c

```diff
@@ -144,17 +147,6 @@ static void send_fp(struct transport_tx *trans, struct link_device *dev)
                               &autopilot.flight_time);
 }
 
-static void send_body_rates_accel(struct transport_tx *trans, struct link_device *dev)
```

Comment:
This was accidentally removed I guess while rebasing

Reply:
sure, my fault